tests/pkg-relative-url/foo/BAR/baz.opf

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Package Documents</dc:coverage>
+    <dc:creator>Dan Lazin</dc:creator>
+    <dc:date>2021-11-12</dc:date>
+    <dc:description>
+    	The manifest refers to an XHTML file in an arbitrary subfolder that is relative to the package's own arbitrary folder.
+      The reading system must be able to find the content.
+    </dc:description>
+    <dc:identifier id="pub-id">pkg-relative-url</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Content in relative subdirectory</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#confreq-rs-pkg-relative-url</meta>
+    <meta property="dcterms:modified">2021-21-12T00:00:00Z</meta>
```

Comment:
This reference is now wrong.

For the RS document, it is not 100% clear where to put the reference, but I suspect it should be bound to the first paragraph of [§3.4 Manifest](https://www.w3.org/TR/epub-rs-33/#sec-pkg-doc-manifest). (We may need an extra id to the paragraph, though).

The "real" target is probably https://www.w3.org/TR/epub-33/#sec-container-iri, though, more exactly its first paragraph.

I would probably add both references.

Reply:
Ah, thanks for catching that — done. The converse references are in https://github.com/w3c/epub-specs/pull/1908.